docs/osquery/osquery.asciidoc

```diff
@@ -61,12 +61,11 @@ TIP: To save a single query for future use, click *Save for later* and define th
 [[osquery-view-history]]
 == View or rerun previous live queries
 
-The *Live queries history* section on the *Live queries* tab shows a log of queries run over the last 30 days.
-Each query has the following options:
+The *Live queries history* section on the *Live queries* tab shows a log of queries run over the last 30 days. From the Live queries table, you can do the following:
 
-* Click image:images/play-icon.png[Right-pointing triangle] to rerun a query.
+* Click image:images/play-icon.png[Right-pointing triangle] to rerun a single query or a query pack.
 
-* Click image:images/table-icon.png[Table icon] to view the query <<osquery-results,results>> and <<osquery-status,status>>.
+* Click image:images/table-icon.png[Table icon] to examine the <<osquery-results,results>> for a single query or a query pack. From the results table, you can also find the query <<osquery-status,status>>.
```

Comment:
Are there names/labels for these icons? It'd be nice to mention them in the text like we've done in Security docs, but Kibana docs might have a different convention for inline images like this.

Reply:
@joepeeples these icons don't have names that display in the UI when you hover over them. @gchaps do you have any recommendations for naming these or objections to naming them?

Reply:
I thought they might not have labels in the UI. In Security docs we've still given a description or unofficial name (example below), which I prefer over including only an image, but @gchaps please let us know what would work best for Kibana docs.

<img width="850" alt="image" src="https://user-images.githubusercontent.com/87339667/187471608-73db3094-4e94-4fe4-bfbb-07cb39556590.png">

Reply:
Our guideline is to use the form above, but we typically use the word icon instead of menu:

>Include the icon inline to reinforce what the user is interacting with. If the icon doesn’t have a name, give it one.

Another guideline we follow for accessibility is to avoid directional language:

> Above, below, left, right, and so on. All these terms assume that the layout never varies and that users actually see it, and that they see it the way you think they do. What about disabilities or responsiveness?